jsk_spot_robot/jsk_spot_startup/launch/rviz.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find spot_viz)/rviz/robot.rviz" />
```

Comment:
should point to `jsk_spot_startup/config/spot.rviz ` ??

Reply:
Fixed the filepath to rviz config.